### PR TITLE
radmin.c: Fix typo

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: src/modules/proto_control/radmin.c [hunk 1]
```diff
@@ -838,6 +838,7 @@ int main(int argc, char **argv)
 
 		uid = getuid();
 		gid = getgid();
+		subcs = NULL;
```
Comment (alandekok) [inline src/modules/proto_control/radmin.c]: This shouldn't be needed.

Comment (jpereira) [reply src/modules/proto_control/radmin.c]: @alandekok It was to mute the clang.

```
src/modules/proto_control/radmin.c:846:12: warning: 2nd function call argument
      is an uninitialized value
  ...subcs = cf_section_find_next(cs, subcs, "server", server);
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
src/modules/proto_control/radmin.c:858:20: warning: 2nd function call argument
      is an uninitialized value
  ...((subcs = cf_section_find_next(cs, subcs, "server", CF_IDENT_ANY)) != NU...
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
2 warnings generated.
Decode with:```

Comment (jpereira) [reply src/modules/proto_control/radmin.c]: https://travis-ci.org/FreeRADIUS/freeradius-server/jobs/549750054

**Comment by alandekok** [comment]
Ah. My mistake. I’ll push a fix.

**Pull request closed without merging**